coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -383,6 +383,19 @@ def test_mv(self, vector_length, backend):
         TorchBaseTest.run_compare_torch((vector1, vector2), model, backend=backend, input_as_shape=False)
 
 
+class TestOuter(TorchBaseTest):
+    @pytest.mark.parametrize("x_vector_length, y_vector_length, backend",
+                             itertools.product([1, 5, 7], [1, 3, 6], backends)
```

Comment:
I will reduce the test configure to `[1, 5], [1, 3]`

Reply:
Thx, I addressed this comment.